src/OrchardCore.Modules/OrchardCore.Authentication/Startup.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Modules;
+
+namespace OrchardCore.Authentication
+{
+    public class Startup : StartupBase
+    {
+        public override void Configure(IApplicationBuilder app, IRouteBuilder routes, IServiceProvider serviceProvider)
+        {
+            app.UseAuthentication();
+        }
+
+        public override void ConfigureServices(IServiceCollection services)
+        {
+            services.AddAuthentication();
+
+            // Note: IAuthenticationSchemeProvider is already registered at the host level.
```

Comment:
Is it making the host one useless then? Do we need it in any other scenario?

Reply:
I suppose developers could still register non-tenant aware handlers at the host level.

Note that this code is not new. It was just moved from the Users module.